### DISCUSSION TITLE
Reasons I love the tailwind OSS project (Thankyou @tailwindlabs)

### DISCUSSION THREAD

**Discussion by benwinding** [tailwindlabs/tailwindcss#4464] [Kind words]
Tailwind has been a key part of my development flow for the past 2 years. Just wanted to share my appreciation for the project to the creators.

Here's the reasons I love `tailwind.css`:
- **Documentation** is simple and very helpful. A lot of work obviously went into, making it; fast, searchable, maintainable and easy to read.
- **Logical** naming of the API, `flex: column` == `flex-col`. This has 2 benefits; it's easy to discover the API if you know CSS, and it helps you actually learn the underlying CSS too!
- **Share-ability** (if that's a word) is amazing. Any tailwind HTML can easily be copied to other projects, just like components, makes software re-use easy and predictable.
- **Integration** is easy and available for many types of projects. It's clear a lot of work went into making adoption of the library easy.
- Finally it simply **improves my life** as a developer. It makes sites simpler and easier to maintain in pretty much all component-based architectures.

Thanks Guys! :heart:

**Comment by simonswiss** [comment]
Thank you for an awesome shoutout, @benwinding  🎉

> This has 2 benefits; it's easy to discover the API if you know CSS, and it helps you actually learn the underlying CSS too!

I really love that in particular ☝️
